community-operators/twistlock/twistlock-console-operator.v0.0.9.clusterserviceversion.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: twistlock-console-operator.v0.0.9
+  namespace: placeholder
+  annotations:
```

Comment:
Please add annotations as outlined in our doc: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md#required-fields-for-operatorhub